### ISSUE TITLE
Extension does not work

### ISSUE THREAD

**Issue by clay-walker** [fivetran/dbt-language-server#1126]
![image](https://github.com/fivetran/dbt-language-server/assets/93723976/d14543dc-c348-4bdd-823c-01994b6ee259)

I receive the above message when generating a preview by clicking the icon in the top right corner.

When I right click on a ref and try any of the wizard commands... nothing happens at all and it fails silently, no error message and no results for any of the menu options.

v0.28.2 installed.

**Comment by clay-walker** [comment]
I noticed that the runtime status contains this "not yet activated" message. How can I activate? 

![image](https://github.com/fivetran/dbt-language-server/assets/93723976/1e669161-9736-4e5f-a24a-0be0306385a6)

**Comment by pgrivachev** [comment]
Hi @clay-walker! 

It is hard to say why extension was not activated without more details. Could you please try [these steps](https://docs.google.com/document/d/e/2PACX-1vQwri6c9d9a4GRQVJR65ur_O_9a9zhY8SZymvZLTgO07ziiypuvC0UCMoO5MIXn61MiTv6DxqqRkzp8/pub) and check if you have any logs from the extension? Also what Operating System do you use and what version of VS Code and dbt?

**Comment by clay-walker** [comment]
@pgrivachev I've already tried those steps, and no luck! Unfortunately no logs. 

I'm on macOS Monterey (Need to update 😬 )
Visual Studio 1.63.2

**Comment by pgrivachev** [comment]
Version of VS Code is too old (November 2021). Is it possible for you to [update it](https://code.visualstudio.com/Docs/setup/setup-overview#_update-cadence)?

**Comment by clay-walker** [comment]
@pgrivachev thanks. wow, didn't realize I was so out of date. 

Updated and it kind of works. The generated preview appears to be the same thing as the dbt model. I'm assuming it's failing to compile to sql.

![image](https://github.com/fivetran/dbt-language-server/assets/93723976/c6bdd8df-d254-47eb-98f9-eb79ef4185cc)

**Comment by clay-walker** [comment]
I got this error message. Note: I do have dbt-snowflake installed and it's on my path (I can open a command prompt and typing "dbt" works).

![image](https://github.com/fivetran/dbt-language-server/assets/93723976/cb71136a-4d96-4326-b40d-efede7a1fc04)

**Comment by pgrivachev** [comment]
@clay-walker The extension tries to find dbt in Python environment specified in VS Code. Did you install dbt using pip? Could you try to specify the Python environment that was used to install dbt using this docs: https://code.visualstudio.com/docs/python/environments#_manually-specify-an-interpreter. 
Also if it doesn't help it will be good if you send me extension logs.

**Comment by esenilsson** [comment]
I have similar issues, where it tries to find bigquery-adapter even though I am using dbt-snowflake adapter

![CleanShot 2023-08-10 at 11 54 14@2x](https://github.com/fivetran/dbt-language-server/assets/2824187/1c81e308-aa5e-4002-95f8-4c89d52b6980)

**Comment by pgrivachev** [comment]
Hey @esenilsson, could you send me [extension logs](https://docs.google.com/document/d/e/2PACX-1vQwri6c9d9a4GRQVJR65ur_O_9a9zhY8SZymvZLTgO07ziiypuvC0UCMoO5MIXn61MiTv6DxqqRkzp8/pub) to investigate the issue?

**Comment by esenilsson** [comment]
```

dbtproj 2023-08-10T09:51:07.727Z:  Starting to recompile/reanalyze the project
dbtproj 2023-08-10T09:51:07.727Z:  Starting project compilation
dbtproj 2023-08-10T09:51:08.749Z:  Process '"/Users/username/Library/Caches/pypoetry/virtualenvs/dbt-hokQvezB-py3.8/bin/python" "/Users/username/.vscode-insiders/extensions/fivetran.dbt-language-server-0.28.2-darwin-arm64/server/python/script.py" 25683 /Users/username/projects/dbt/dbtproj/ ~/.dbt compile ' exited with code 1
dbtproj 2023-08-10T09:51:08.749Z:  There was an error while project compilation Command failed: "/Users/username/Library/Caches/pypoetry/virtualenvs/dbt-hokQvezB-py3.8/bin/python" "/Users/username/.vscode-insiders/extensions/fivetran.dbt-language-server-0.28.2-darwin-arm64/server/python/script.py" 25683 /Users/username/projects/dbt/dbtproj/ ~/.dbt compile 
Traceback (most recent call last):
  File "/Users/username/.vscode-insiders/extensions/fivetran.dbt-language-server-0.28.2-darwin-arm64/server/python/script.py", line 7, in <module>
    from dbt.adapters.bigquery import BigQueryColumn
ModuleNotFoundError: No module named 'dbt.adapters.bigquery'

dbtproj 2023-08-10T10:57:15.095Z:  Process '"/Users/username/Library/Caches/pypoetry/virtualenvs/dbt-hokQvezB-py3.8/bin/python" "/Users/username/.vscode-insiders/extensions/fivetran.dbt-language-server-0.28.2-darwin-arm64/server/python/script.py" 25683 /Users/username/projects/dbt/dbtproj/ ~/.dbt compile -m +models/snowflake/crm/crm_activationtype.sql' exited with code 1
dbtproj 2023-08-10T10:57:15.370Z:  dbt compilation error: Command failed: "/Users/username/Library/Caches/pypoetry/virtualenvs/dbt-hokQvezB-py3.8/bin/python" "/Users/username/.vscode-insiders/extensions/fivetran.dbt-language-server-0.28.2-darwin-arm64/server/python/script.py" 25683 /Users/username/projects/dbt/dbtproj/ ~/.dbt compile -m +models/snowflake/crm/crm_activationtype.sql
Traceback (most recent call last):
  File "/Users/username/.vscode-insiders/extensions/fivetran.dbt-language-server-0.28.2-darwin-arm64/server/python/script.py", line 7, in <module>
    from dbt.adapters.bigquery import BigQueryColumn
ModuleNotFoundError: No module named 'dbt.adapters.bigquery'
```

**Comment by pgrivachev** [comment]
@esenilsson thank you! I was able to reproduce the issue on my side and now working on fix.

**Comment by pgrivachev** [comment]
Hey folks, I published new extension version v0.29.0 with fix. Could you please update extension and check if fix helps?
